chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts

```diff
@@ -1692,12 +1692,12 @@ export const dummyDefaultProps: ILiveChatWidgetProps = {
         },
         localizedTexts: {
             MIDDLEWARE_BANNER_FILE_NULL_ERROR: "There was an error uploading the file, please try again.",
-            MIDDLEWARE_BANNER_FILE_SIZE_WITHOUT_EXTENSION_ERROR: "File exceeds the allowed limit of {0} MB and please upload the file with an appropriate file extension.",
-            MIDDLEWARE_BANNER_FILE_SIZE_EXTENSION_ERROR: "File exceeds the allowed limit of {0} MB and {1} files are not supported.",
-            MIDDLEWARE_BANNER_FILE_WITHOUT_EXTENSION: "File upload error. Please upload the file with an appropriate file extension.",
+            MIDDLEWARE_BANNER_FILE_SIZE_WITHOUT_EXTENSION_ERROR: "File {0} exceeds the allowed limit of {1} MB and please upload the file with an appropriate file extension.",
```

Comment:
By adding {0} backward compatibility is broken. We need to clearly log this change in the Change Log, and also keep supporting the old format.